File: EBiletJava/app/src/main/java/com/alperensarac/ebiletjava/data/model/Venue.java
package com.alperensarac.ebiletjava.data.model;

import com.google.gson.annotations.SerializedName;

/*
    Venue.java

    Sahne / mekan modelidir.
*/
public class Venue {

    private int id;

    @SerializedName("city_id")
    private Integer cityId;

    @SerializedName("district_id")
    private Integer districtId;

    private String name;

    private String address;

    private Integer capacity;

    public int getId() {
        return id;
    }

    public Integer getCityId() {
        return cityId;
    }

    public Integer getDistrictId() {
        return districtId;
    }

    public String getName() {
        return name;
    }

    public String getAddress() {
        return address;
    }

    public Integer getCapacity() {
        return capacity;
    }
}

File: EBiletJetpack/app/src/main/java/com/alperensarac/ebiletjetpack/data/model/Venue.kt
package com.alperensarac.ebiletjetpack.data.model

import com.google.gson.annotations.SerializedName

/*
    Sahne / mekan modeli.
*/
data class Venue(
    val id: Int,

    @SerializedName("city_id")
    val cityId: Int? = null,

    @SerializedName("district_id")
    val districtId: Int? = null,

    val name: String,

    val address: String? = null,

    val capacity: Int? = null
)